catalog/plugin/pom.xml

```diff
@@ -33,5 +33,6 @@
         <module>catalog-plugin-expirationdate</module>
         <module>catalog-plugin-content-uri</module>
         <module>catalog-plugin-clientinfo</module>
+        <module>catalog-plugin-metacardingest-network</module>
```

Comment:
Conditional statements are based off of network data values from the client (re: servlet API): 

**See:**
`remoteAddr`
`remoteHost`
`scheme`
`contextPath`

See [ServletRequest, Java EE 6](http://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html)